static/app/views/releases/list/releaseAdoptionChart.tsx

```diff
@@ -200,23 +204,27 @@ class ReleaseAdoptionChart extends React.PureComponent<Props, State> {
               />
             )}
           </ChartZoom>
-        </ChartBody>
-        {
-          <ChartFooter>
-            {tct('Total [display] [count]', {
-              display: activeDisplay === DisplayOption.USERS ? 'Users' : 'Sessions',
-              count: <Count value={get24hCountByProject ?? 0} />,
-            })}
-          </ChartFooter>
-        }
+        </PanelBody>
+        <ChartFooter>
+          <InlineContainer>
+            <SectionHeading>
+              {tct('Total [display]', {
+                display: activeDisplay === DisplayOption.USERS ? 'Users' : 'Sessions',
+              })}
+            </SectionHeading>
+            <SectionValue>
+              <Count value={totalCount ?? 0} />
+            </SectionValue>
+          </InlineContainer>
```

Comment:
Placeholder size needs adjusting because there is jumping now when going from loading to loaded state.

Reply:
Yes, the footer was a bit short increased the placeholder to 34px.